### PR TITLE
Update README for accuracy, especially "features"

### LINKED ISSUE #1211: Split "features" section of README into core, addons, and goals
[The "features" header](http://docs.mantl.io/en/latest/#features) of our documentation is confusing. It contains a lot of bullet points, some for addons, some for core components, and some for... adjectives. [This list is duplicated (imperfectly) in our README](https://github.com/CiscoCloud/mantl/tree/d8b7e9d9e935c03b65b22ba4e1f3e2b641731738#features), which is where it actually belongs. 

We should remove it from the other documentation and split it up into three sections. Note that this list organizes components according to our plans going forward, not the current reality (notably, Logstash and Collectd are in `sample.yml`): 
 1. Core components
* [Consul](http://consul.io) for service discovery
* [Vault](http://vaultproject.io) for managing secrets
* [Mesos](http://mesos.apache.org) cluster manager for efficient resource isolation and sharing across distributed services
* [Marathon](https://mesosphere.github.io/marathon) for cluster management of long running containerized services
* [Terraform](https://terraform.io) deployment to multiple cloud providers
* [Docker](http://docker.io) container runtime
* [Traefik](https://traefik.github.io/) for proxying external traffic
* [mesos-consul](https://github.com/CiscoCloud/mesos-consul) populating Consul service discovery with Mesos tasks
* [marathon-consul](https://github.com/CiscoCloud/marathon-consul) update consul k/v with Marathon tasks
* [Mantl API](https://github.com/CiscoCloud/mantl-api) easily install supported Mesos frameworks on Mantl
* [Mantl UI](https://github.com/CiscoCloud/nginx-mantlui) a beautiful administrative interface to Mantl
    mesos-consul populating Consul service discovery with Mesos tasks
    marathon-consul update consul k/v with Marathon tasks
 2. Addons
* [ELK Stack](https://www.elastic.co/webinars/introduction-elk-stack) for log collection and analysis
     -  [Logstash](https://github.com/elastic/logstash) for log forwarding
* [GlusterFS](http://www.gluster.org/) for container volume storage
* [etcd](https://github.com/coreos/etcd) distributed key-value store for Calico
* [Calico](http://www.projectcalico.org) a new kind of virtual network
* [collectd](https://collectd.org/) for metrics collection
 3. Goals
* Security
* High availability
* Multi-datacenter support

### PR DESCRIPTION
**Pull request by siddharthist**
* Removes duplicated information both within the README and between README and other docs
 * Removes rename notice
 * Updates list of features with current status
 * Updates tagged versions

Fixes #1211

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,41 +38,68 @@ Mantl is a modern, batteries included platform for rapidly deploying globally di
...
-* [marathon-consul](https://github.com/CiscoCloud/marathon-consul) update consul k/v with Marathon tasks
-* [GlusterFS](http://www.gluster.org/) for container volume storage
 * [Traefik](https://traefik.github.io/) for proxying external traffic
-* [Mantl API](https://github.com/CiscoCloud/mantl-api) easily install supported Mesos frameworks on Mantl
-* [Mantl UI](https://github.com/CiscoCloud/nginx-mantlui) a beautiful administrative interface to Mantl
-* Multi-datacenter support
-* High availability
+* [mesos-consul](https://github.com/CiscoCloud/mesos-consul) populating Consul
+  service discovery with Mesos tasks
+* [marathon-consul](https://github.com/CiscoCloud/marathon-consul) update
```
Comment (ryane) [inline README.md]: marathon-consul was removed a while back: https://github.com/CiscoCloud/mantl/commit/a3c35b4614731b25f16d9144db580301da936e05

File: README.md [hunk 2]
```diff
@@ -40,41 +38,68 @@ Mantl is a modern, batteries included platform for rapidly deploying globally di
...
+* [GlusterFS](http://www.gluster.org/) for container volume storage
+* [etcd](https://github.com/coreos/etcd) distributed key-value store for Calico
+* [Calico](http://www.projectcalico.org) a new kind of virtual network
+* [collectd](https://collectd.org/) for metrics collection
+* [Kong](http://getkong.org) for managing APIs
+
+### Goals
 * Security
+* High availability
+* Multi-datacenter support
```
Comment (ryane) [inline README.md]: do we actually still have Multi-datacenter support?

File: README.md [hunk 3]
```diff
@@ -40,41 +38,68 @@ Mantl is a modern, batteries included platform for rapidly deploying globally di
...
 
-The base platform contains control nodes that manage the cluster and any number of resource nodes. Containers automatically register themselves into DNS so that other services can locate them.
+The base platform contains control nodes that manage the cluster and any number
+of resource nodes. Containers automatically register themselves into DNS so
+that other services can locate them.
 
 ![Single-DC](docs/_static/single_dc.png)
 
-Once WAN joining is configured, each cluster can locate services in other data centers via DNS or the [Consul API](http://www.consul.io/docs/agent/http.html).
+Once WAN joining is configured, each cluster can locate services in other data
```
Comment (ryane) [inline README.md]: maybe we should just remove all of the multi-dc stuff for now

**Comment by siddharthist** [comment]
Thanks for the review @ryane, it's rebased and fixed!

**Pull request merged**